hasura/graphql-engine
# fix nested errors array in ws-server response log

### Description


`errors` field in `ws-server` response log is duplicated nested payload

```json
{
  "type": "ws-server",
  "timestamp": "2020-02-18T14:48:34.949+0700",
  "level": "debug",
  "detail": {
    "event": {
      "type": "message_sent",
      "detail": "{
        \"type\":\"data\",
        \"id\":\"1\",
        \"payload\":{
          \"data\":null,
          \"errors\":[{
            \"errors\":[{
              \"extensions\":{
                \"path\":\"$\",
                \"code\":\"unexpected\"
              },
              \"message\":\"postgres query error\"
            }]
          }]
        }
      }"
    },
    "websocket_id": "277c227f-12c1-46a8-8763-b6c205962f78"
  }
}
```
### Affected components



### Related Issues




### Solution and Design


Use wrong typo `encodeGQErr` function 

### Steps to test and verify


- Start HGE with log level `debug`
- Subscribe a success subscription
- Modify table structure (e.g rename column)
- The subscription is alive, but response debug `ws-server` error
- See stdout logs

### Limitations, known bugs & workarounds



### Server checklist


#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL
   
   

#### Breaking changes


  1. Metadata API

     Existing `query` types:
     

  2. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  3. Logging

     





*Note*: This isn't really a breaking change, because the log payload is incorrect. I'm not sure if anyone use this log, since it is on debug level.
